packages/components/nodes/embeddings/AWSBedrockEmbedding/AWSBedrockEmbedding.ts

```diff
@@ -144,7 +162,10 @@ class AWSBedrockEmbedding_Embeddings implements INode {
             if (iModel.startsWith('cohere')) {
                 return await embedTextCohere(documents, client, iModel, inputType)
             } else {
-                return Promise.all(documents.map((document) => embedTextTitan(document, client, iModel)))
+                // return Promise.all(documents.map((document) => embedTextTitan(document, client, iModel)))
```

Comment:
if we have proven the new approach works, we can go ahead and delete this

Reply:
I did test on my setup using large Confluence space (2500+ pages before split) using flow below. Token Text Splitter is a must, otherwise you get to many tokens error, not related to this issue. It takes some time to process, but result is stored to vector database without errors, mentioned in opened issue #2907. I also noticed, I forgot to delete commented line, sorry for that.

<img width="1709" alt="AWSBedrockEmbeddingsFlow" src="https://github.com/user-attachments/assets/e6ca29b7-0fd4-4020-b02d-e679213ff1cc">
<img width="1725" alt="AWSBedrockEmbeddingsFlow2" src="https://github.com/user-attachments/assets/b25e2ccb-c9a9-4ee7-a8a2-d55ab9581094">

Reply:
yeah we can go ahead and remove the commented line

Reply:
Done.